plugins/modules/inventory_intent.py

```diff
@@ -2245,37 +2314,38 @@ def get_diff_merged(self, config):
             device_ids = self.get_device_ids(device_ips)
 
             if len(device_ids) == 0:
-                self.msg = "Can't Assign Global User Defined Field to device as device's are not present in Cisco DNA Center"
                 self.status = "failed"
+                self.msg = "Can't Assign Global User Defined Field to device as device's are not present in Cisco Catalyst Center"
+                self.log(self.msg, "INFO")
                 self.result['changed'] = False
                 return self
 
             # Now add code for adding Global UDF to device with Id
             self.add_field_to_devices(device_ids).check_return_status()
 
             self.result['changed'] = True
-            self.msg = "Global User Defined Added with name {0} added to device Successfully !".format(field_name)
-            self.log(self.msg)
+            self.msg = "Global User Defined Added with name '{0}' added to device Successfully !".format(field_name)
```

Comment:
"Global User Defined Fields (UDF) named '{0}' have been successfully added to the device.".format(field_name)
or
"Global User Defined Field (UDF) named '{0}' has been successfully added to the device.".format(field_name)

Reply:
added.